.github/workflows/build-ros2-ws-action.yml

```
@@ -0,0 +1,82 @@
+name: ros2-ws-action
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [master, feature/fix-github-workflows]
```

I suggests removing the `feature/fix-github-workflows` branch from triggering the workflow. Any branch other than the main branch can be triggered by the `pull_request` event instead.
```suggestion
  push:
    branches: [master]
```